src/pyscaffoldext/dsproject/templates/readme_md.template

```diff
@@ -18,23 +18,23 @@ In order to set up the necessary environment:
    ```
    conda activate ${name}
    ```
-3. install `${name}` with:
-   ```
-   python setup.py install # or `develop`
-   ```
+
+> **_NOTE:_**  The conda environment will have ${name} installed in editable mode.
+> Some changes, e.g. in `setup.cfg`, might require you to run `pip install -e .` again.
```

Comment:
> `pip install -e .`
👍

Nice I like replacing `python setup.py develop/install` with a direct call to `pip` because I think it is more future-proof...
In my mind, direct usage of `setup.py` is going to be less and less frequently until it is completely deprecated...

Probably we are going to see more "compiling-related" projects (like cython) to tap into the PEP517 specs and provide plugins to be specified inside `pyproject.toml`, leaving the packaging files purely declarative...
(I just wonder how setuptools' maintainers are going to reconcile these trends... If, for example, I am using cython, I don't want to completely replace my `build-backend` in `pyproject.toml`, I want to keep setuptools but just augment it for some files....)